src/crawler.js

```diff
@@ -112,7 +137,7 @@ Crawler.prototype.crawl = function(ipp, hops) {
     self.dequeue(ipp);
 
     if (err) {
-      self.logger.error(ipp + ' has err ', err);
+      //self.logger.error(ipp + ' has err ', err);
```

Comment:
Its best to strip out commented code or document why it is commented out, since future hackers will read this and wonder why it was commented out in the first place.